### PR TITLE
DOCS-1552: Update documentation of IMM Metrics

### PR DESCRIPTION
**Pull request by mbechtold-splunk**
Expanded from DOCS-2113. Add two metrics to metrics.yaml:

- sf.org.subscription.function
- sf.org.numResourceMetrics

### CODE REVIEW

File: signalfx-org-metrics/metrics.yaml [hunk 1]
```diff
@@ -1290,6 +1290,17 @@ sf.org.numPropertyLimitedMetricTimeSeriesCreateCallsByToken:
   metric_type: counter
   title: sf.org.numPropertyLimitedMetricTimeSeriesCreateCallsByToken
 
+sf.org.numResourceMetrics:
```
Comment (jmalin-signalfx) [inline signalfx-org-metrics/metrics.yaml:1293]: Does our user documentation already have a definition of "Resource metric time series"? I just added a PR to explain the terms "active" MTS and "inactive MTS", and I anticipate that someone is going to ask for the same explanation of Resource metric time series.

File: signalfx-org-metrics/metrics.yaml [hunk 2]
```diff
@@ -1290,6 +1290,17 @@ sf.org.numPropertyLimitedMetricTimeSeriesCreateCallsByToken:
...
+  description: |
+    Number of resource metric time series (MTS) received from host and container 
+    categories. 
+    
+    * Dimension(s): `orgId`, `resourceType`
+    * Data resolution: 10 minutes
+  metric_type: gauge
+  title: sf.org.numResourceMetrics
+
 sf.org.numResourcesMonitored:
```
Comment (jmalin-signalfx) [inline signalfx-org-metrics/metrics.yaml:1304]: What does "Resources" include?

File: signalfx-org-metrics/metrics.yaml [hunk 3]
```diff
@@ -1666,25 +1677,25 @@ sf.org.log.numContentBytesReceivedByToken:
...
-    Number of containers.
+    Number of containers included in the subscription.
     
     * Data resolution: 10 seconds
   metric_type: gauge
   title: sf.org.subscription.containers
 
 sf.org.subscription.customMetrics:
-  brief: Number of custom metric time series.
+  brief: Number of custom metric time series (MTS).
```
Comment (jmalin-signalfx) [inline signalfx-org-metrics/metrics.yaml:1687]: I think we have a definition for "custom metric time series", but please check.

File: signalfx-org-metrics/metrics.yaml [hunk 4]
```diff
@@ -1693,8 +1704,18 @@ sf.org.subscription.highResolutionMetrics:
...
   brief: Number of hosts.
   description: |
-    Number of hosts.
+    Number of hosts included in the subscription.
     
       * Data resolution: 10 seconds
   metric_type: gauge
   title: sf.org.subscription.hosts
+ 
+sf.org.subscription.function:
```
Comment (jmalin-signalfx) [inline signalfx-org-metrics/metrics.yaml:1713]: Do we already define "serverless function"?

**Review by jmalin-signalfx — COMMENTED** [review]
See my comments. Mostly I want to provide additional information before someone comes looking for it.

File: signalfx-org-metrics/metrics.yaml (hunk 1 above)
Comment (mbechtold-splunk) [reply signalfx-org-metrics/metrics.yaml:1293]: Yes -- a "resource MTS" represents the use of subscription resources like custom MTS.

File: signalfx-org-metrics/metrics.yaml (hunk 4 above)
Comment (mbechtold-splunk) [reply signalfx-org-metrics/metrics.yaml:1713]: Good question. "Serverless function" is detailed in the docs site, but seems to refer only to AWS Lambda functions.

**Pull request merged**